### PR TITLE
docs: update asyncapi document definition

### PR DESCRIPTION
**Pull request by alequetzalli**
The original spec text seems to inaccurately suggest that multiple AsyncAPI documents are required for each application, whereas the correct statement would be that a single AsyncAPI document, potentially referencing other files, is typically used to describe an application's API.

Perhaps it is more accurate to say that the AsyncAPI Specification defines a set of fields that can be used in one or more AsyncAPI documents to describe an application's API. A single AsyncAPI document can reference other files, but it doesn't mean that multiple AsyncAPI documents are required for a single application.

While the AsyncAPI document is the primary file describing the API, it can include references to other files that contain additional details or shared fields. This doesn't equate to requiring multiple AsyncAPI documents for a single application.

### related to https://github.com/asyncapi/website/pull/1868/files#r1424279213

### CODE REVIEW

**Review by derberg — COMMENTED** [review]
@fmvilas @smoya @dalelane @char0n hey folks, please have a look, especially related discussion, to make sure we are all aligned

File: spec/asyncapi.md [hunk 1]
```diff
@@ -14,8 +14,7 @@ The AsyncAPI Specification is licensed under [The Apache License, Version 2.0](h
 
 The AsyncAPI Specification is a project used to describe message-driven APIs in a machine-readable format. It’s protocol-agnostic, so you can use it for APIs that work over any protocol (e.g., AMQP, MQTT, WebSockets, Kafka, STOMP, HTTP, Mercure, etc).
 
-The AsyncAPI Specification defines a set of files required to describe the API of an [application](#definitionsApplication).
-These files can be used to create utilities, such as documentation, code, integration, or testing tools.
+The AsyncAPI Specification defines a set of fields that can be used in an AsyncAPI document to describe an [application](#definitionsApplication)'s API. The document may reference other files for additional details or shared fields, but it is typically a single, primary document that encapsulates the API description.
 
 The file(s) SHOULD describe the operations an [application](#definitionsApplication) performs. For instance, consider the following AsyncAPI definition snippet:
```
Comment (derberg) [inline spec/asyncapi.md:19]: lgtm, just maybe we should get rid of this `file` reference as well, so whenever possible, we are consistently saying `AsyncAPI document`

Comment (alequetzalli) [inline spec/asyncapi.md:19]:
```suggestion
The AsyncAPI document SHOULD describe the operations an [application](#definitionsApplication) performs. For instance, consider the following AsyncAPI definition snippet:
```

Comment (alequetzalli) [reply spec/asyncapi.md]: done!

**Review by smoya — APPROVED** [review]
Nice catch @alequetzalli! This looks far more accurate now.

**Comment by alequetzalli** [comment]
> Nice catch @alequetzalli! This looks far more accurate now.

@smoya well thank youuuuu 😎

**Review by derberg — APPROVED** [review]
week passed, no more comments, merging

Thanks @alequetzalli !

**Comment by smoya** [comment]
@all-contributors please add @alequetzalli for content,doc,example,ideas,review

**Pull request merged**